packages/accounts/src/smart_account/mod.rs

```diff
@@ -307,7 +311,7 @@ pub enum SmartAccountError {
     /// The specified context rule does not exist.
     ContextRuleNotFound = 2000,
     /// A conflicting context rule already exists.
```

Comment:
should we also change the error explanation here, and use `duplicate` instead of `conflicting`?